libs/designer/src/lib/ui/settings/parameters.tsx

```diff
@@ -0,0 +1,93 @@
+import { updateParameterConditionalVisibility } from '../../core/state/operation/operationMetadataSlice';
+import type { Settings } from './settingsection';
```

Comment:
Please dont call it parameters, let's keep it in settings concept for now and add like searchable setting or just a normal utility inside some other folder in designer-ui...
When we do refactoring if we do then we can separate the concepts

Reply:
Sure. You're talking about just the file name + component name, right?

Reply:
Actually I see what you're driving at. I can make this more broadly usable.

Reply:
Okay, I've separated the component logic best I can into the `designer-ui` library.